src/libPMacc/include/algorithms/math/floatMath/trigo.tpp

```diff
@@ -83,12 +84,23 @@ struct Sinc<float>
 {
     typedef float result;
 
-    HDINLINE double operator( )(const float& value )
+    HDINLINE float operator( )(const float& value )
     {
       if(::fabsf(value) < FLT_EPSILON)
-	return 1.0;
+    return 1.0;
       else
-	return ::sinf( value )/value;
+    return ::sinf( value )/value;
+    }
+};
+
+template<>
+struct Atan2<float>
+{
+    typedef float result;
+
+    HDINLINE float operator( )(const float& val1, const float& val2 )
```

Comment:
pls use `( const float val1, const float val2 )`: byRef does not make sense for integral types